### DISCUSSION TITLE
New pages deployed successfully, but the content is not updated

### DISCUSSION THREAD

**Discussion by polyrabbit** [community/community#70100] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Dears, 

I have a site that is deployed by github action. But today I found the content of that site is not updated even though `actions/deploy-pages@v2` reports a successful deployment.

```
Artifact exchange URL: https://pipelinesghubeus13.actions.githubusercontent.com/w1AVt3OYjL5qLlgHjhuakL4BlcpqxLoufRLc8ookZ27u3qWs8Z/_apis/pipelines/workflows/6518343007/artifacts?api-version=6.0-preview
Creating Pages deployment with payload:
{
	"artifact_url": "https://pipelinesghubeus13.actions.githubusercontent.com/w1AVt3OYjL5qLlgHjhuakL4BlcpqxLoufRLc8ookZ[27](https://github.com/polyrabbit/hacker-news-digest/actions/runs/6518343007/job/17703669651#step:16:28)u3qWs8Z/_apis/pipelines/1/runs/51886/artifacts?artifactName=github-pages&%24expand=SignedContent",
	"pages_build_version": "f513d4b01884044035634bdbec0c4382d1a16b65",
	"oidc_token": "***"
}
Created deployment for f513d4b01884044035634bdbec0c4382d1a16b65, ID: f513d4b01884044035634bdbec0c4382d1a16b65
Getting Pages deployment status...
Reported success!
```

Is there any way I can diagnose issue, e.g. logs/status page?

Thanks in advance. One successful github action can be found here: https://github.com/polyrabbit/hacker-news-digest/actions/runs/6518343007/job/17703669651

**Comment by Mayurrajput97** [comment]
you can Double-check that the GitHub Actions workflow is deploying from the correct branch. It's possible that it might be deploying from a different branch than the one you're expecting.

**Reply by polyrabbit** [reply]
I use `actions/deploy-pages@v2` to do the deployment, which deploys an artifact created by a prior `actions/upload-pages-artifact@v2`, who uses a local directory as the input. So it's not deploying from a certain branch.

The weird thing is - I didn't change anything, and but the content won't update since yesterday just out of sudden.
